charts/kube-prometheus-stack/Chart.yaml

```
@@ -22,7 +22,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 42.0.2
+version: 42.0.3
```

Ease raise minor version.
```suggestion
version: 42.1.0
```